homeassistant/components/light/lifx.py

```diff
@@ -227,6 +227,25 @@ def turn_on(self, **kwargs):
         self._liffylights.set_color(self._ip, hue, saturation,
                                     brightness, kelvin, fade)
 
+    def set_brightness(self, **kwargs):
```

Comment:
Hmm, this is not consistent with the turn off method which takes no args, but I'll make the change.